src/database/migrations/0020-create-users.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE users (
```

Comment:
Flagging this file: I think I have to rename the migration once @hminsky2002 merges his validation PR

Reply:
(done!)